docs/core/testing/unit-testing-mstest-sdk.md

```diff
@@ -36,75 +39,108 @@ When you `build` the project all the needed components will be restored and inst
+You can set the profile using the property `TestingExtensionsProfile` to one of the following 3 profiles:
 
-To enable a specific profile, use the `TestingExtensionsProfile` MSBuild property.  
-For example, if you wish to enable all Microsoft testing extensions, you can configure the `TestingExtensionsProfile` as shown below:
+- `None` - no extension is added,
+- `Default` - the default profile when none is explicitly set. Depending on the version of the SDK, the recommended list of extensions.
+- `AllMicrosoft` - enable all extensions shipped by Microsoft (including extensions with a restrictive license).
+
+Here is a full example, using the `None` profile:
 
 ```xml
-<Project Sdk="MSTest.Sdk/3.3.0">
+<Project Sdk="MSTest.Sdk/3.3.1">
+
     <PropertyGroup> 
         <TargetFramework>net8.0</TargetFramework>
-        <TestingExtensionsProfile>AllMicrosoft</TestingExtensionsProfile>
+        <TestingExtensionsProfile>None</TestingExtensionsProfile>
     </PropertyGroup>    
+
     <!-- references to the code to test -->
+
 </Project>
 ```
 
-> [!NOTE]
-> If you don't specify a custom `TestingExtensionsProfile`, the `Default` profile will be applied.
+### Enabling or disabling extensions
 
-If you prefer not to use the default profile, you can set it to `None`:
+On top of the extensions profile, we offer the ability to enable or disable extensions using a MSBuild property with the pattern `Enable[NugetPackageNameWithoutDots]`.  
+
+For example, to enable the crash dump extension (NuGet package [Microsoft.Testing.Extensions.CrashDump](https://www.nuget.org/packages/Microsoft.Testing.Extensions.CrashDump)), you can use the following property `EnableMicrosoftTestingExtensionsCrashDump` set to `true`:
 
 ```xml
-<Project Sdk="MSTest.Sdk/3.3.0">
-    <PropertyGroup> 
-        <TargetFramework>net8.0</TargetFramework>
-        <TestingExtensionsProfile>None</TestingExtensionsProfile>
-    </PropertyGroup>    
-    <!-- references to the code to test -->
+<Project Sdk="MSTest.Sdk/3.3.1">
+
+<PropertyGroup> 
+    <TargetFramework>net8.0</TargetFramework>
+    <EnableMicrosoftTestingExtensionsCrashDump>true</EnableMicrosoftTestingExtensionsCrashDump>
+</PropertyGroup>
+
+<!-- references to the code to test -->
+
 </Project>
 ```
 
-You can always opt-out specific extension setting to false its respective MSBuild property.  
-For instance you can opt-out the default `MS Code Coverage` by setting `<EnableMicrosoftTestingExtensionsCodeCoverage>false</EnableMicrosoftTestingExtensionsCodeCoverage>`. Full-example below:
+This page lists all [extensions](./unit-testing-platform-extensions.md) available.
+
+> [!WARNING]
+> Please review the licensing terms for each extension as they may vary.
+
+You can combine the profiles with this feature. For example, you can opt-out from the `Default` profile the `MS Code Coverage` extension by setting `<EnableMicrosoftTestingExtensionsCodeCoverage>false</EnableMicrosoftTestingExtensionsCodeCoverage>`. Full-example below:
```

Comment:
```suggestion
Enabled and disabled extensions are combined with the extensions provided by your selected extension profile.

This can be used to enable an additional extension on top of the implicit `Default` profile (as seen in the CrashDumpExtension example above). 

Or to disable an extension that is coming from the selected profile. In this case disabling `MS Code Coverage` extension by setting `<EnableMicrosoftTestingExtensionsCodeCoverage>false</EnableMicrosoftTestingExtensionsCodeCoverage>`:
```